scipy/stats/_stats_py.py

```diff
@@ -7742,52 +7743,15 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
 
     Examples
     --------
```

Comment:
I see the refernence in "See also" above, and that looks like a good idea. Can we also add a link at the end of the example? E.g., "For a more detailed example, see..."?

Reply:
Will do 👍🏻